pkg/kubernetes/scripts/projects.js

```diff
@@ -641,6 +655,41 @@
         }
     ])
 
+    .controller("ProjectDeleteCtrl", [
+        "$scope",
+        "$modalInstance",
+        "dialogData",
+        "kubeMethods",
+        "$location",
+        //"projectActions",
+        function($scope, $instance, dialogData, methods, $location) {
+            angular.extend($scope, dialogData);
+            //angular.extend($scope, actions);
+
+
+            $scope.performDelete = function performDelete() {
+                return methods.delete(dialogData.project);
+            };
+            console.debug($scope.performDelete)
+            console.debug($scope)
+            if ($scope.complete) {
```

Comment:
You would do this above in performDelete. See here for an example:

https://github.com/cockpit-project/cockpit/blob/master/pkg/kubernetes/scripts/images.js#L172